pythonbits/bb.py

```diff
@@ -425,14 +446,25 @@ def _render_resolution(self):
     def _render_additional(self):
         additional = []
         audio_tracks = self['tracks']['audio']
+        file_path = self['path']
         text_tracks = self['tracks']['text']
 
+        # assumes the user:
+        # 1) has a folder structure that contains 'remux' in the path
+        #    (ex. /mnt/movies/remux/filename.ext)
+        # 2) the file is properly named, containing 'remux'
+        if any(x in file_path for x in ('remux', 'Remux', 'REMUX')):
+            if 'REMUX' not in additional:
+                additional.append('REMUX')
+
         for track in audio_tracks[1:]:
             if 'title' in track and 'commentary' in track['title'].lower():
-                additional.append('w. Commentary')
+                if 'w. Commentary' not in additional:
+                    additional.append('w. Commentary')
 
         if text_tracks:
-            additional.append('w. Subtitles')
+            if 'w. Subtitles' not in additional:
+                additional.append('w. Subtitles')
```

Comment:
Why do you think this change is necessary?

Reply:
Recently a submission was presented with 2-3 instances of w. Subtitles or w. Commentary, I can't quite remember, so this will prevent that. Which then had to be updated manually.

Please let me know if there is an inherantly easier way to accomplish this.

Reply:
I don't really see how that could ever happen. `additional` is defined only in the function scope, so 'w. Subtitles' can only ever be added once.